content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -42,6 +47,58 @@ to only enforce the policy on non-development workspaces is more appropriate.
 [run-glossary]: /docs/glossary.html#run
 [workspace-glossary]: /docs/glossary.html#workspace
 
+## Namespace: root
+
+The **root namespace** contains data associated with the current run.
+
+### Value: `created_at`
+
+* **Value Type:** String.
+
+The `created_at` value within the [root namespace](#root-namespace) specifies the time that the run was created.
+
+### Value: `message`
+
+* **Value Type:** String.
+
+The `message` value within the [root namespace](#root-namespace) specifies the message that is associated with the run.
+
+The default value is *"Queued manually via the Terraform Enterprise API"*.
+
+### Value: `speculative`
+
+* **Value Type:** Boolean.
+
+The `speculative` value within the [root namespace](#root-namespace) specifies whether the plan associated with the run is a [speculative plan](../../run/index.html#speculative-plans) only.
+
+### Value: `is_destroy`
+
+* **Value Type:** Boolean.
+
+The `is_destroy` value within the [root namespace](#root-namespace) specifies whether the run will perform a destroy operation.
+
+### Value: `variables`
```

Comment:
I'm a little torn on this one. The nested nature of it makes it seem less like an attribute and more like a namespace. Are we clear on when we think about things as a namespace vs a nested set of attributes?

Reply:
@glenngillen I originally had variables listed as a separate namespace but @chrisarcand commented in TF-199 and requested that I change it to an attribute instead.